### ISSUE TITLE
Unexpected error when try to signin

### ISSUE THREAD

**Issue by dutch-erik** [PimDoos/onesmartcontrolha#8]
First of all, I’m very happy with this integration! Thank you!

However, after upgrade to latest version 0.1.0 I’m to able to log in anymore. 
Error is: unexpected error.

```
This error originated from a custom integration.

Logger: custom_components.onesmartcontrol.config_flow
Source: custom_components/onesmartcontrol/config_flow.py:48 
Integration: One Smart Control (documentation, issues) 
First occurred: 20:53:16 (3 occurrences) 
Last logged: 20:56:02

Unexpected exception
Traceback (most recent call last):
  File "/config/custom_components/onesmartcontrol/config_flow.py", line 78, in async_step_user
    info = await validate_input(self.hass, user_input)
  File "/config/custom_components/onesmartcontrol/config_flow.py", line 48, in validate_input
    connection_status = await wrapper.connect()
TypeError: OneSmartWrapper.connect() missing 1 required positional argument: 'socket_name'

```
I’ve been using previous version as well, but removed it because after some time the processor load increase (doubled) and therefore RPI temp increased with almost 20C. After deleting the OSC integration every went back to normal.
So with this new version I wanted to check if that was solved or not. It’s not possible due to the above unexpected error.

**Comment by PimDoos** [comment]
I'm looking into this, looks like a regression from v0.1.0. I have a patch which should fix this ready to deploy in a few minutes :)

Regarding the CPU usage, this probably has to do with the the poll runner. I'll see if I can tweak the performance on that a bit as well.

**Comment by PimDoos** [comment]
Should be fixed in [v0.1.1](https://github.com/PimDoos/onesmartcontrolha/releases/tag/v0.1.1)

**Comment by dutch-erik** [comment]
After below warnings, processor load increased.
After the first increase I restarted, but as you can see, it came back after some time.

```
Logger: root
Source: custom_components/onesmartcontrol/onesmartwrapper.py:177
Integration: One Smart Control (documentation, issues)
First occurred: 13:01:01 (12 occurrences)
Last logged: 15:44:43

Could not update '['boiler_flow_temperature', 'water_tank_temperature', 'return_temperature', 'flow_temperature']' for 'HEATPUMP': {'error': 3, 'description': 'timeout'}
Could not update '['heat_source_status', 'seven_segment_display_error_code', 'refrigerant_error_info', 'defrost']' for 'HEATPUMP': {'error': 3, 'description': 'timeout'}
Could not update '['outlet_air_percentage', 'inlet_air_percentage']' for 'COMFOAIR': {'error': 3, 'description': 'timeout'}
Could not update '['error_a', 'co2_level_first_floor', 'co2_level_ground_floor', 'exit_air_temperature']' for 'COMFOAIR': {'error': 3, 'description': 'timeout'}
Command timed out after 60 seconds: apparatus
```
```
Logger: root
Source: custom_components/onesmartcontrol/onesmartwrapper.py:189
Integration: One Smart Control (documentation, issues)
First occurred: 15:44:43 (3 occurrences)
Last logged: 15:44:46

Command timed out after 60 seconds: ping
Ping to server timed out. Reconnecting.
Connection error on socket push: '[Errno 113] Host is unreachable' Reconnecting in 60 seconds.
```
![afbeelding](https://user-images.githubusercontent.com/62548287/187231572-d07c8a2f-b248-4e79-a8a9-c434e8a072b2.png)
